app/models/school.rb

```diff
@@ -878,6 +878,89 @@ def pupil_numbers_ok?
     end
   end
 
+  def self.from_onboarding(onboarding)
+    sch = new
+    sch.data_enabled =                    false
+    sch.name =                            onboarding.school_name
+    sch.urn =                             onboarding.urn
+    sch.establishment_id =                onboarding.urn
+
+    return sch if sch.establishment.nil?
+
+    sch.establishment =                   sch.establishment.current_establishment
+    sch.urn =                             sch.establishment_id
+
+    est = sch.establishment
+    sch.name =                            est.establishment_name
+    sch.address =                         address_from_establishment(est)
+    sch.postcode =                        est.postcode
+    sch.website =                         est.school_website
+    sch.school_type =                     school_type_from_phase_of_education_code(est.phase_of_education_code)
+    sch.number_of_pupils =                est.number_of_pupils
+    sch.percentage_free_school_meals =    est.percentage_fsm
+    sch.region =                          region_from_gor_code(est.gor_code)
```

Comment:
looking at the data I see they're using a gor_code of "W" to indicate Wales. Could you also update the `country` field to be Wales in this case, otherwise England?